dogfooding/lib/screens/stats_latency_chart.dart

```diff
@@ -96,14 +96,16 @@ class StatsLatencyChart extends StatelessWidget {
                   colors: [
                     ColorTween(
                             begin: AppColorPalette.appGreen,
-                            end: AppColorPalette.appGreen.withOpacity(0.5))
+                            end:
+                                AppColorPalette.appGreen.withValues(alpha: 0.5))
                         .lerp(0.2)!
-                        .withOpacity(0.1),
+                        .withValues(alpha: 0.1),
```

Comment:
Also, the alpha now ranges from (0 - 255), so we also have to multiply it by 255
```suggestion
			.withValues(alpha: 255.0 * 0.1)
```

Reply:
I guess, it's better to ignore the deprecation at this stage and we can replace it in the next major upgrade.

Reply:
I also just now figured out that it will fail for 3.24, so I reverted back to `withOpacity()` 😅 
But you don't have to multiply by 255, `withValues(alpha: )` works with doubles just like withOpacity() and not like withAlpha() - tested this.